<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusBar</class>
 <widget class="QStatusBar" name="StatusBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QSlider" name="ZoomSlider">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>5</y>
     <width>170</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider {
background-color: rgba(1, 1, 1, 0);
margin: 0 0;
}</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="sliderPosition">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
   <property name="tickInterval">
    <number>100</number>
   </property>
  </widget>
  <widget class="QToolButton" name="ZoomMinus">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>5</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
background-color: transparent;
}

QToolButton:hover {
background-color: rgba(224, 224, 224, 196);
border: 0px;
}

QToolButton:pressed {
background-color: rgba(135, 135, 135, 196);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="ZoomPlus">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>5</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
background-color: transparent;
}

QToolButton:hover {
background-color: rgba(224, 224, 224, 196);
border: 0px;
}

QToolButton:pressed {
background-color: rgba(135, 135, 135, 196);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="ZoomNumber">
   <property name="geometry">
    <rect>
     <x>536</x>
     <y>5</y>
     <width>30</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
background-color: transparent;
}

QToolButton:hover {
background-color: rgba(224, 224, 224, 196);
border: 0px;
}

QToolButton:pressed {
background-color: rgba(135, 135, 135, 196);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
  </widget>
  <widget class="QLabel" name="drawName">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>5</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Shape: Line</string>
   </property>
  </widget>
  <widget class="QLabel" name="pos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>X: 0000 Y: 0000</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
